### ISSUE TITLE
Log connections

### ISSUE THREAD

**Issue by SassNinja** [mistakster/throttle-proxy#21]
When I first tried the proxy I wasn't completely sure if it's really working because I didn't get any feedback in the command prompt.
Something option like `--verbose` to show if a connection is going though the proxy would be great!

Can this be added?

**Comment by mistakster** [comment]
Hi! Yes, it can. I hope I make the changes soon.

**Comment by mistakster** [comment]
@SassNinja, actually you can enable debug information using environment variable `DEBUG` like this:

```
DEBUG=throttle-proxy:* throttle-proxy
```

or like this:

```
DEBUG=* throttle-proxy
```

Anyway, I agree that the explicit option will be more useful.

**Comment by mistakster** [comment]
I’ve just published v2.3.0 with `-v` option.

**Comment by SassNinja** [comment]
@mistakster many thanks for this super fast release!

Works great 👍
